test/functional/fixtures/shell-test.c

```diff
@@ -121,6 +121,7 @@ int main(int argc, char **argv)
       fprintf(stderr, "Unknown first argument: %s\n", argv[1]);
       return 3;
     }
+    wait();  // Flush stdout.
```

Comment:
Thanks.

So this is not done implicitly on exit?!

Reply:
I don't know.

Reply:
Me neither - but would expect otherwise.
Also I've thought that `wait(3)` would only be relevant / do something with childs.

Reply:
`wait()` is a local static function. I'll rename it.

Reply:
Oh.. :D